spec/master_lock/redis_lock_spec.rb

```diff
@@ -31,11 +29,6 @@
     described_class.new(config: cluster_backend.config, key: "key", owner: "owner4", ttl: 0.1)
   end
 
-  before do
-    clean_redis
```

Comment:
do we still need to do this?

Reply:
yes but it turns out we also clean from inside RSpec.configure in spec_helper so the ones I am removing were redundant.